src/stories/packages/components/Select.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { useArgs } from '@storybook/preview-api';
+import type { Meta, StoryFn } from '@storybook/react';
+import { Select } from '@terra-ui-packages/components';
+import React from 'react';
+
+const meta: Meta<typeof Select> = {
+  title: 'Packages/Components/Select',
+  component: Select,
+  parameters: {
+    layout: 'centered',
```

Comment:
The dropdown gets cut off on the Docs page. You can force a larger area for the component on the docs page by specifying within the parameters object:

```    
docs: {
      story: {
        height: '200px',
      },
    },